src/components/ContactInfo/ContactInfo.vue

```diff
@@ -277,6 +275,10 @@ export default class ContactInfo extends Vue {
   @Prop({ default: false })
   private disableActions!: boolean
 
+  /** Option to disable the action tooltip. */
+  @Prop({ default: false })
+  private disableActionTooltip!: boolean
```

Comment:
Ack, double negative! I had to think through this one, but I believe it's correct: by default, disabled is false, which means the tooltip will be enabled (same as prev code).

Reply:
Good point, i didn't look at it from that angle, i can understand the confusion!

Reply:
Easy review = great code

Hard review = maybe the reviewer is tired 😆

Reply:
Thank you! 

I hear ya, i feel tired more often than not lately